brother_ql/backends/helpers.py

```diff
@@ -63,9 +64,6 @@ def send(instructions, printer_identifier=None, backend_identifier=None, blockin
 
     if not blocking:
         return status
-    if selected_backend == 'network':
-        """ No need to wait for completion. The network backend doesn't support readback. """
-        return status
```

Comment:
can you expand on this change?

Reply:
These lines are removed, because the network backend now does support readback. 
Therefore we can continue executing the function and fill in and return the actual status values.